mmv1/templates/terraform/examples/org_policy_custom_constraint_basic.tf.erb

```diff
@@ -0,0 +1,13 @@
+resource "google_org_policy_custom_constraint" "<%= ctx[:primary_resource_id] %>" {
+  provider = google-beta
+
+  name         = "custom.disableGkeAutoUpgrade"
+  parent       = "organizations/%{org_id}"
```

Comment:
```suggestion
  parent       = "organizations/<%= ctx[:test_env_vars]['org_id'] %>"
```

I think we want to use `<%= ctx[:test_env_vars]['org_id'] %>` rather than `%{org_id}` here & below, see https://github.com/GoogleCloudPlatform/magic-modules/blob/5268d03a79ba5644c9b8252175f642245881c8b7/mmv1/templates/terraform/examples/firebase_project_basic.tf.erb#L6 for a prior example.

This injects the same thing into the tests, but generates a nicer value in the docs: https://github.com/hashicorp/terraform-provider-google-beta/blob/d75d53a18d3b01c4c29257babcaa351e014bae9b/google-beta/resource_firebase_project_generated_test.go#L54 + https://github.com/hashicorp/terraform-provider-google-beta/blame/d75d53a18d3b01c4c29257babcaa351e014bae9b/website/docs/r/firebase_project.html.markdown#L47

Reply:
Ah ok, I definitely did not understand how that piece was working. I updated the examples, and thanks for explaining.